src/com/lftechnology/java/training/alina/basic/multiplesum/SumOfMultiples.java

```diff
@@ -4,50 +4,46 @@
 import java.util.logging.Logger;
 
 /**
- * If we list all the natural numbers below 10 that are multiples of 3 or 5, we
- * get 3, 5, 6 and 9. The sum of these multiples is 23. Find the sum of all the
- * multiples of 3 or 5 below 1000.
+ * If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23. Find
+ * the sum of all the multiples of 3 or 5 below 1000.
  * 
  * @author Alina Shakya<alinashakya@lftechnology.com>
  */
 public class SumOfMultiples {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)